bokehjs/src/coffee/models/plots/gmap_plot_canvas.coffee

```diff
@@ -156,16 +156,13 @@ class GMapPlotCanvasView extends PlotCanvas.View
     ctx.lineTo(left,    top)
     ctx.closePath()
 
-    ctx.fillStyle = @mget('border_fill_color')
+    ctx.fillStyle = @model.plot.border_fill_color
     ctx.fill()
 
 class GMapPlotCanvas extends PlotCanvas.Model
   type: 'GMapPlotCanvas'
   default_view: GMapPlotCanvasView
 
-  @define {
-      map_options: [ p.Any ]
```

Comment:
Oh never mind. They were already there, which is why they didn't show up in the diff. Sorry for the noise.